content/events/Zama-meetup-afterwork-create-the-future-of-end-to-end-encrypted-blockchain.md

```diff
@@ -0,0 +1,22 @@
+---
+eventName: "Zama Meetup & Afterwork: Let's Create the Future of End-to-end Encrypted Blockchain"
+eventDescription: "This meetup is organized by the team at Zama, as a side event to the European Blockchain Convention."
+eventTopic: 
+  - "Blockchain"
+  - "Web3"
+  - "FHE"
+  - "Privacy"
+eventType: "Social"
+location: "Barcelona, Spain"
+venueAddress: "​Palau Dalmases"
+venueMapsLink: "https://maps.app.goo.gl/RV85fP4qJnBtvUXv5"
+website: "https://lu.ma/zama-meetup-labweek23"
+startDate: "10/25/2023"
+endDate: "10/25/2023"
+tag: "Side event"
```

Comment:
@zaccherinij An event can have either of the two options as its tag 'PLN Event' or 'Industry Event'. Please update this accordingly.

Reply:
Done, thanks!